src/pages/api/login.ts

```diff
@@ -19,16 +19,18 @@ export default async function handleUserInfo(
       ? await bcrypt.compare(userPassword, userInfo.userpassword)
       : undefined;
 
-    if (rightPassword) {
+    if (userInfo && rightPassword) {
       const cookies = new Cookies(req, res);
       const ENC_KEY =
-        process.env.ENC_KEY || "default_key_for_risback_cinema_hello_there";
+        process.env.ENC_KEY || 'default_key_for_risback_cinema_hello_there';
 
       cookies.set(
-        "session",
+        'session',
         await Iron.seal(
           {
             username: userName,
+            firstName: userInfo.firstName,
```

Comment:
aha I see that you sent user's name through cookie! Nice :) I guess it is encoded, so it won't be any risk that showing users information in cookie in the end? nice! 🥇